AK/QuickSelect.h

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
(GitHub still doesn't support commenting on the commit message, so I'll just write it here)

collection* (lowercase)
option*
time* complexity*
useful*